src/util/uid.js

```diff
@@ -6,9 +6,10 @@
  * Legal characters for the unique ID.
  * Should be all on a US keyboard.  No XML special characters or control codes.
  * Removed $ due to issue 251.
+ * Removed _ which denotes word separation in XML.
  * @private
  */
-const soup_ = '!#%()*+,-./:;=?@[]^_`{|}~' +
```

Comment:
I don't think this change is required anymore.

Reply:
Done, thanks!